crates/chainspec/src/spec.rs

```diff
@@ -317,13 +317,6 @@ impl ChainSpec {
 
     /// Get the initial base fee of the genesis block.
     pub fn initial_base_fee(&self) -> Option<u64> {
-        // TODO(scroll): migrate to Chain::scroll() (introduced in https://github.com/alloy-rs/chains/pull/112) when alloy-chains is bumped to version 0.1.48
-        if self.chain == Chain::from_named(NamedChain::Scroll) ||
-            self.chain == Chain::from_named(NamedChain::ScrollSepolia)
-        {
-            return None
-        }
-
```

Comment:
Is this not required because the `base_fee_per_gas` is provided in genesis?

Reply:
This was a quick fix in order to retrieve the correct `base_fee_per_gas` for `Scroll` even though `London` is activated at block 0. However, with the introduction of the `make_genesis_header `, which allows for better control and readability this is indeed not needed anymore.